main.css

```diff
@@ -0,0 +1,1153 @@
+/* Extra small devices (phones, 600px and down) */
```

Comment:
Me when I open this file:

<img width="86" alt="image" src="https://user-images.githubusercontent.com/43992068/167295831-ee4f776c-1784-4f52-819a-6cc5350181ce.png">


You're right. Copying of styles for different screen sizes make sense only when styles differ significantly.

This is how it looks like now:

<img width="310" alt="image" src="https://user-images.githubusercontent.com/43992068/167295945-1c12063e-a2c1-454c-97e3-52325abce1e3.png">

Instead of trying to adapt block sizes for all screens, you can just use proper `display: flex` containers with `flex-wrap` property to ensure you will always provide enough space for items on your page. For now I won't be checking this whole file because you probable need to remove all styles for different screens and start from the scratch.

https://doka.guide/css/flexbox-guide/ this might be of help to you.